### PR TITLE
Add readme file for image redactor

### PR DESCRIPTION
**Pull request by shiranr**
Add readme file

### CODE REVIEW

**Review by biancafbmd — APPROVED** [review]

File: presidio-image-redactor/README.MD [hunk 1]
```diff
@@ -1,18 +1,101 @@
...
--   Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract) by following the instructions on how to install it for your operating system. The Image-Redactor was tested with a minimal version of 4.0.0
--   Install [Spacy model](https://spacy.io/usage/models) to use with the presidio-analyzer.
+- Install [Tesseract OCR](https://github.com/tesseract-ocr/tesseract) by following the
+  instructions on how to install it for your operating system.
+
+  The Image-Redactor was tested with a minimal version of 4.0.0
+- Install [Spacy model](https://spacy.io/usage/models) to use with the
+  presidio-analyzer.
+
+  Once Installed, run the following command to download the default spacy model needed
```
Comment (omri374) [inline presidio-image-redactor/README.MD:21]: Once the image-redact or is installed with pip, the user will have spacy installed, so no other action besides the model download is necessary.

File: presidio-image-redactor/README.MD [hunk 2]
```diff
@@ -1,18 +1,101 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in alpha and not production ready.***
+
 ## Description
 
-The Presidio Image Redactor is a Python based module for detecting and redacting PII text
-entities in images.
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
```
Comment (omri374) [inline presidio-image-redactor/README.MD:7]: Maybe add the flow diagram from the ADR?

File: presidio-image-redactor/README.MD (hunk 1 above)
Comment (shiranr) [reply presidio-image-redactor/README.MD:21]: Done.

File: presidio-image-redactor/README.MD (hunk 2 above)
Comment (shiranr) [reply presidio-image-redactor/README.MD:7]: Done.

**Review by omri374 — APPROVED** [review]

**Comment by omri374** [comment]
Looks great, thanks!

**Pull request merged**